docs/actions-pipelines.md

```diff
@@ -36,12 +36,10 @@ expectations:
 actions:
 
   generate_study_population:
-    run: cohortextractor:latest generate_cohort --study-definition study_definition
+    run: cohortextractor:latest generate_cohort --study-definition study_definition --output-format csv.gz
```

Comment:
thought: I'm guessing we can't change this to be the default, because it could break things for existing users?

(Which is also why the lack of image versioning is a slight pain here, but is fixed in Data Builder.)

Can we encourage existing users to go via this compressed CSV route somehow? Maybe even adding PRs to the studies that use the most space?

Reply:
Yes we can't change it.

There's plan to warn if you are using it.